driver/src/main/scala/api/collections/GenericCollection.scala

```diff
@@ -622,6 +622,14 @@ trait GenericCollection[P <: SerializationPack with Singleton] extends Collectio
       Future.failed(MissingMetadata())
   }
 
+  def deleteOne[S](selector: S, writeConcern: WriteConcern = defaultWriteConcern)(implicit swriter: pack.Writer[S], ec: ExecutionContext): Future[WriteResult] = {
```

Comment:
I prefer not to introduce those aliases for `delete.{one,many}`

Reply:
please note that `def deleteMany` is not an alias for `delete.many`, but an alias for `delete.one` without a limit on the documents to delete. There is no alias for `delete.many`.
Having to call `delete().one(selector, limit = None)` to delete all (i.e many) documents that match the selector seems a bit unintuitive to me... 
If you still think that the best is to not having these methods, I will gladly remove them and had them in my project as extension methods...

Reply:
:ok_hand:

Reply:
So these could be removed for now. Thx

Reply:
removed